src/Microsoft.Azure.SignalR.Common/Utilities/HttpClientFactory.cs

```diff
@@ -8,21 +8,17 @@ namespace Microsoft.Azure.SignalR
 {
     internal class HttpClientFactory
     {
-        internal static IHttpClientFactory ClientFactory { get; }
+        private readonly static IHttpClientFactory _httpClientFactory;
 
         static HttpClientFactory()
         {
             var serviceCollection = new ServiceCollection();
             serviceCollection.AddHttpClient();
             var services = serviceCollection.BuildServiceProvider();
 
-            ClientFactory = services.GetRequiredService<IHttpClientFactory>();
-        }
-
-        public static HttpClient CreateClient()
-        {
-            return ClientFactory.CreateClient();
+            _httpClientFactory = services.GetRequiredService<IHttpClientFactory>();
         }
 
+        public static HttpClient Client => _httpClientFactory.CreateClient();
```

Comment:
I think the previous `CreateClient()` is better, the semantic of property sounds more like some value that `get` costs not expensive and rarely change